shared/src/popup.css

```diff
@@ -64,6 +64,7 @@ input[type="password"]:focus {
 .logo {
   width: 58px;
   height: 29px;
+  margin-top: 10px;
```

Comment:
This just gives the extension popup a more symmetrical look, vertically, as the bottom has a bit of extra spacing.